todo/src/AddTodoForm.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function AddTodoForm(props) {
+
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const todoTitle = event.target.title.value;
+    console.log(todoTitle);
```

Comment:
nitpik: You can remove this`console.log` now. You're showing the `newTodo` via above the todo list now :)